tree/ntuple/v7/inc/ROOT/RNTupleDescriptor.hxx

```diff
@@ -443,6 +442,8 @@ private:
    std::unique_ptr<RHeaderExtension> fHeaderExtension;
 
 public:
+   static constexpr unsigned int kFeatureFlagTest = 137; // Bit reserved for forward-compatibility testing
```

Comment:
Sorry, could you remind me of the choice of both the value of the feature flag and why it's defined inside this class body?

Reply:
The value is arbitrary (but larger than 64 to test chains of feature flag integers). There is no real feature behind this feature flag. It's reserved to test reacting to (unknown) feature flags in the reader.